doc/changes/changes_0.1.0.md

```diff
@@ -7,7 +7,7 @@ In this first release we created an docker test environment for Exasol. The test
 ### Currently supported Operating Systems
 
 * Linux
-* Mac OS X with Intel processors (please consult the [README](../../README.md) for more details)
```

Comment:
dead link causes a warning in `myst` (which is treated as error. Haven't discovered how to suppress this kind of warning. So I removed the link....